src/pybind/mgr/dashboard/frontend/src/app/shared/api/auth.service.ts

```diff
@@ -13,19 +13,19 @@ export class AuthService {
 
   login(credentials: Credentials) {
     return this.http
-      .post('api/auth', credentials)
+      .post('api/auth',
+            {username: credentials.username, password: credentials.password})
       .toPromise()
-      .then((resp: Credentials) => {
+      .then((resp: any) => {
```

Comment:
Should we add `token` to `Credentials` class? That way we can keep using that type in the response.

Reply:
I would prefer not to do that because the information require to do the login is different from the information returned by a login operation. What about creating a class called `LoginResponse` for this?